docs/get_started/tour.mdx

```diff
@@ -45,1015 +51,828 @@ This guide is written for:
+            2024-04-16 17:33:59 INFO  [CartObject/addTicket][inv_1aiqX0vFEFNH5uLBb8M6CjbRkVUcVScH1T] dev.restate.sdk.core.InvocationStateMachine - End invocation
+            // withClass highlight-line
+            2024-04-16 17:33:59 DEBUG [CartObject/addTicket] dev.restate.sdk.http.vertx.RequestHttpServerHandler - Handling request to CartObject/addTicket
+            2024-04-16 17:33:59 INFO  [CartObject/addTicket] dev.restate.sdk.core.ResolvedEndpointHandlerImpl - Start processing invocation
+            2024-04-16 17:33:59 DEBUG [CartObject/addTicket][inv_1aiqX0vFEFNH5uLBb8M6CjbRkVUcVScH1T] dev.restate.sdk.core.InvocationStateMachine - Transitioning state machine to REPLAYING
+            // withClass highlight-line
+            2024-04-16 17:33:59 WARN  [CartObject/addTicket][inv_1aiqX0vFEFNH5uLBb8M6CjbRkVUcVScH1T] dev.restate.sdk.core.ResolvedEndpointHandlerImpl - Error when processing the invocation
+            java.lang.IllegalStateException: The handler failed
+            ... rest of trace ...
+            2024-04-16 17:33:59 WARN  [CartObject/addTicket][inv_1aiqX0vFEFNH5uLBb8M6CjbRkVUcVScH1T] dev.restate.sdk.core.InvocationStateMachine - Invocation failed
+            java.lang.IllegalStateException: The handler failed
+            ... rest of trace ...
+            ```
+
+        </details>
+    </TabItem>
 </Tabs>
 
-Send an `addTicket` request, as [you did earlier](#running-the-services-and-runtime).
-Now you see in the logs that both services get suspended.
-The user session service gets suspended because it waits for a response from the `reserve` function and the ticket service gets suspended because it waits for the sleep to be completed.
-Restate keeps track of how long the service should sleep and then triggers it to resume the invocation.
-Finally, you see the responses of both functions coming in.
+Call `CartObject/addTicket` again and have a look at the service logs.
+You see the retries taking place. And you see that only the first time the call to the `CheckoutService` was made.
+The other times, the call was skipped and the journaled response was replayed.
+
+[By default](/operate/configuration/#default-configuration), Restate will keep retrying failed invocations until they succeed.
+If you want to cancel an invocation in a retry loop, you can use the CLI to do this. Let's have a look at that next.
+
+## Debugging with the CLI
+
+Now that we have a failing invocation, let's take the opportunity to show you how you can get more information about what is going on with the CLI.
+The CLI is a management tool that lets you interact with the Restate Server.
+You can use it to boostrap a new project, but also to get information about the services and invocations.
+
+Have a look at some useful commands and try them out yourself:
+
+<CliAnimation listOfSteps={[`$ restate services list
```

Comment:
This works super well with arrows! But there is a little number on the side of it, and feel noisy and unnecessary.